postgres/Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG POSTGRES_BASE_IMAGE=postgres:10.12
```

Comment:
this is different from what's in `Makefile` - does that matter? is there a way to have a single source of truth for these versions?